### PR TITLE
start script

### PR DESCRIPTION
**Pull request by peterpt**
This pull request is just an idea , if you want to commit then go ahead , it is your choice .
compiled chaos is good for those that have issues with golang compiler configuration , but if you update your code then you need to recompile chaos again or users will still run the old version .

The start script basically avoid the user to run on the terminal "go run CHAOS.go" , with this script all that users have to write in terminal window on CHAOS directory is : ./start

this way the user will always run the updated version of CHAOS as long as user frequently gets a git checkout , (ex: git pull) to update his version on his OS with your latest changes .

However , this start script will not solve the problem if user have his go compiler not configured or is still in an old version .

Note : if you merge this pull request , then user does not have to write execution permissions on "start" script , because script already have those permissions on github and will stay there if you merge with your git .

Basically all user have to do is to clone your git , enter chaos directory and execute ./start .

### CODE REVIEW

**Comment by tiagorlampert** [comment]
Really is more easy to users run `./start`.
But i will keep the compiled version of CHAOS on the directory, and update it frequently when code was changed. To user run `./CHAOS`.

**Comment by tiagorlampert** [comment]
Hi, i removed the compiled version, 'cause someone open a [issue](https://github.com/tiagorlampert/CHAOS/issues/9). The problem was caused because even running the compiled version, it is necessary have golang package installed to compile the code.
If you want, send me again a pull request with _start_ file. Or i will create the file _start_.
